dnf-behave-tests/features/download-source.feature

```diff
@@ -0,0 +1,93 @@
+Feature: Tests for picking the appropriate download source
+
+
+@not.with_os=rhel__eq__8
+@bz1775184
+Scenario: baseurl is used if all mirrors from mirrorlist fail
+Given I create directory "/baseurlrepo"
+  And I execute "createrepo_c /{context.dnf.installroot}/baseurlrepo"
```

Comment:
Nitpick: the `/` in `/{context.dnf.installroot}/baseurlrepo` is superfluous. (in all scenarios)

Reply:
:+1: